doc/api/path.md

```diff
@@ -330,6 +330,15 @@ For example on POSIX:
 path.normalize('/foo/bar//baz/asdf/quux/..');
 // Returns: '/foo/bar/baz/asdf'
 ```
+*Note*: The `path.posix.normalize()` method will not attempt to convert `\ ` 
```

Comment:
This note is backwards.

The docs clearly state normalize does two things: resolving . and .. segments, and collapsing multiple path seperators.  Adding a statement about what it does NOT do is just strange.

The real problem is that the docs are wrong. On Windows, both `/` and `\` are valid directory seperators (the docs are wrong here), and all sequences of *1 or more* valid path seperator are replaced with a single *preferred* path seperator.

For POSIX, there is only one path seperator, so the set of valid and preferred is identical.

Fow Windows, it nees a special note, there are two valid seperators, but one is preferred, so seperators may actually be changed during normalize:

```
> path.win32.normalize("hi////there\\\\/\\/\\\/you/there")
'hi\\there\\you\\there'
```

Besides being documented, it would also be useful to have a couple examples.